content/graphql/guides/forming-calls-with-graphql.md

```diff
@@ -43,6 +51,8 @@ read:public_key
 read:gpg_key
 ```
 
+{% endif %}
+
```

Comment:
@skedwards88 said:

> Thanks for this suggestion! We recommend these scopes to match the scopes granted to the GraphQL explorer. The explorer is not currently available for enterprise users, so for them we just display `The following scopes are recommended:` instead of `To match the behavior of the GraphQL Explorer, request the following scopes:`. I would recommend either changing the text that we display for enterprise users, or only removing the `repo` scope recommendation for enterprise users.
> 
> If you are new to the liquid versioning: `{% ifversion fpt or ghec %}` precedes the text that only displays for free/pro/team or GHEC users. `{% else %}` precedes the text that displays for other users.

———

@skedwards88 thanks for the feedback and context! Is the updated diff what you had in mind?